### ISSUE TITLE
Only a single tab gets rendered

### ISSUE THREAD

**Issue by gunnartorfis** [okwasniewski/react-native-bottom-tabs#85]
Not sure how to describe this, just tried v0.1.0 on Expo 52 and the new arch. 

I have three tabs and only one of them is being rendered. They all work but only if I remove the other two.

```tsx
const Tabs = createNativeBottomTabNavigator<MainTabParamList>();

export const MainTab: React.FC = () => {
  return (
    <Tabs.Navigator>
      <Tabs.Screen
        name={Routes.HomeStack}
        component={HomeStack}
        options={{
          title: 'Home',
          tabBarIcon: () => ({ sfSymbol: 'house' }),
        }}
      />
      <Tabs.Screen
        name={Routes.SpeedyBookingStack}
        component={SpeedyBookingStack}
        options={{
          title: 'Test',
          tabBarIcon: () => ({ sfSymbol: 'house' }),
        }}
      />
      <Tabs.Screen
        name={Routes.DiscoverStack}
        component={DiscoverStack}
        options={{
          title: 'Test',
          tabBarIcon: () => ({ sfSymbol: 'house' }),
        }}
      />
    </Tabs.Navigator>
  );
};
```

The components behind the Tabs screens are all Stack navigator (created with `createNativeStackNavigator`). 

Here are relevant dependencies:
```
"@react-navigation/bottom-tabs": "^6.6.1",
"@react-navigation/elements": "^1.3.31",
"@react-navigation/native": "^6.1.18",
"@react-navigation/native-stack": "^6.11.0",
"@react-navigation/stack": "^6.4.1",
"expo": "^52.0.0-preview.5",
"react-native": "0.76.0",
"react-native-bottom-tabs": "^0.1.0",
```

**Comment by okwasniewski** [comment]
Hey, thanks for opening the issue. 

I've published version 0.1.2 which should work with the new architecture, previously there were some issues. 

Just tried `"expo": "~52.0.0-preview.7"` with it and it's working as expected with new architecture enabled. I noticed some issues with Android on SDK 52 preview but they are not happening on 51, will investigate further but it may be related to expo itself

Can you try upgrading?

**Comment by gunnartorfis** [comment]
Yep, working! 👏
